Python/Product/TestAdapter.Executor/Common/PythonTestDiscoverer.cs

```diff
@@ -19,34 +19,10 @@
 using System.Linq;
 using Microsoft.PythonTools.TestAdapter.Config;
 using Microsoft.PythonTools.TestAdapter.Services;
-using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter;
 using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
 
 namespace Microsoft.PythonTools.TestAdapter {
-
-    /// <summary>
-    /// Note even though we specify  [DefaultExecutorUri(PythonConstants.UnitTestExecutorUriString)] we still get all .py source files
-    /// from all testcontainers.  
-    /// </summary>
-    [FileExtension(".py")]
-    [DefaultExecutorUri(PythonConstants.UnitTestExecutorUriString)]
-    public class UnitTestDiscoverer : PythonTestDiscoverer {
-        public UnitTestDiscoverer() : base(TestFrameworkType.UnitTest) {
-        }
-    }
-
-    /// <summary>
-    /// Note even though we specify  [DefaultExecutorUri(PythonConstants.PytestExecutorUriString)] we still get all .py source files
-    /// from all testcontainers.  
-    /// </summary>
-    [FileExtension(".py")]
-    [DefaultExecutorUri(PythonConstants.PytestExecutorUriString)]
-    public class PytestTestDiscoverer : PythonTestDiscoverer {
-        public PytestTestDiscoverer() : base(TestFrameworkType.Pytest) { 
-        }
-    }
-    
```

Comment:
No longer export theses.  Now we export the children of this abstract classs (PytestTestDiscoverer and UnitTestTestDiscoverer)